### PR TITLE
change names of pacemaker 2.0 daemons

### PR DESCRIPTION
**Pull request by taroth21**
(as listed in https://wiki.clusterlabs.org/wiki/Pacemaker_2.0_Daemon_Changes).

@tomschr @gao-yan: Please review, this needs to be merged within the next days to be included in the translation drop.

### CODE REVIEW

File: xml/ha_concepts.xml [hunk 1]
```diff
@@ -742,9 +742,9 @@
     Based on the information in the CIB, the PE then computes the ideal
     state of the cluster and how it should be achieved. The PE feeds a list of
```
Comment (gao-yan) [inline xml/ha_concepts.xml]: Figured there're references of "PE" which stands for "pengine" :-) Now it's renamed to pacemaker-schedulerd.

File: xml/ha_concepts.xml [hunk 2]
```diff
@@ -758,12 +758,12 @@
    <para>
     In some cases, it may be necessary to power off nodes to protect shared
     data or complete resource recovery. For this Pacemaker comes with a
-    fencing subsystem, stonithd. &stonith; is an acronym for <quote>Shoot
+    fencing subsystem, pacemaker-fenced. &stonith; is an acronym for <quote>Shoot
     The Other Node In The Head</quote> and is usually implemented with a
```
Comment (gao-yan) [inline xml/ha_concepts.xml]: It's no longer such an acronym now although I personally like the old name :-)

File: xml/ha_fencing.xml [hunk 3]
```diff
@@ -181,18 +181,18 @@
...
+       pacemaker-fenced is a daemon which can be accessed by local processes or over
        the network. It accepts the commands which correspond to fencing
        operations: reset, power-off, and power-on. It can also check the
        status of the fencing device.
       </para>
       <para>
-       The stonithd daemon runs on every node in the CRM HA cluster. The
-       stonithd instance running on the DC node receives a fencing request
-       from the CRM. It is up to this and other stonithd programs to carry
+       The pacemaker-fenced daemon runs on every node in the CRM HA cluster. The
```
Comment (gao-yan) [inline xml/ha_fencing.xml]: We could drop "CRM" in here.

File: xml/ha_fencing.xml [hunk 4]
```diff
@@ -181,18 +181,18 @@
...
+       pacemaker-fenced instance running on the DC node receives a fencing request
+       from the CRM. It is up to this and other pacemaker-fenced programs to carry
```
Comment (gao-yan) [inline xml/ha_fencing.xml]: ... receives a fencing request from the pacemaker-controld.

File: xml/ha_glossary.xml [hunk 5]
```diff
@@ -10,7 +10,7 @@
  * types of resources: primitives, groups, etc.
  * meta attributes, instance attributes, operations, utilization attributes
  * transition graph
- * daemons: attrd, lrmd, mgmtd, stonithd
+ * daemons: pacemaker-attrd, pacemaker-execd, mgmtd, pacemaker-fenced
```
Comment (gao-yan) [inline xml/ha_glossary.xml]: BTW, mgmtd only existed in SLE11.

File: xml/ha_concepts.xml (hunk 1 above)
Comment (taroth21) [reply xml/ha_concepts.xml]: Yan, does this mean that we need to change any instance of PE or policy engine in the docs? I was of the impression that only the daemon name changed, but that 'policy engine' as a component is still a valid term that we can continue to use. See also https://www.suse.com/documentation/sle-ha-15/book_sleha_guide/data/sec_ha_architecture.html#sec_ha_architecture_layers.

File: xml/ha_fencing.xml (hunk 3 above)
Comment (taroth21) [reply xml/ha_fencing.xml]: Done.

File: xml/ha_concepts.xml (hunk 1 above)
Comment (gao-yan) [reply xml/ha_concepts.xml]: Indeed the old terms are still used here and there, but eventually we'd probably better use the consistent names avoiding confusions :-)
https://clusterlabs.org/pacemaker/doc/en-US/Pacemaker/2.0/html/Pacemaker_Explained/_pacemaker_architecture.html

File: xml/ha_glossary.xml (hunk 5 above)
Comment (taroth21) [reply xml/ha_glossary.xml]: Ah, removed from the comment. That's why I did not find any hits for this term when grepping for it yesterday. ;)

File: xml/ha_concepts.xml (hunk 1 above)
Comment (taroth21) [reply xml/ha_concepts.xml]: Ok, I will remove PE and policy engine from our docs then. This means I also need to adjust the graphics (and the overview text).

Comment (gao-yan) [reply xml/ha_concepts.xml]: In that case, we'd probably better as well rename the other components:

CRM (Cluster Resource Manager) -> pacemaker-controld
LRM (Local Resource Manager) -> pacemaker-execd

Comment (taroth21) [reply xml/ha_concepts.xml]: @gao-yan: I will address all of the above in this comment in a separate PR.

**Pull request merged**